src/plasmapy/utils/decorators/checks.py

```diff
@@ -706,7 +704,7 @@ def _get_unit_checks(  # noqa: C901, PLR0912, PLR0915
             # ensure equivalences are properly formatted
             if _equivs is None or _equivs == [None]:
                 _equivs = None
-            elif isinstance(_equivs, Equivalency):
+            elif isinstance(_equivs, u.Equivalency):
```

Comment:
Astropy wants this imported from the units namespace per astropy#16513, so this now complies with that